bigbluebutton-html5/imports/ui/components/audio/device-selector/component.jsx

```diff
@@ -46,7 +46,7 @@ class DeviceSelector extends Component {
   }
 
   handleEnumerateDevicesError(error) {
-    console.error(error);
+    logClient("info",{ error: reason, method: "handleEnumerateDevicesError" });
```

Comment:
What other things are going to be logged this way? Does the server/server logs really need to know that an individual user had a microphone issue?

It appears this error will not print anywhere on the client either? I'd say for a client hardware error it's probably more important that it gets seen by the user rather than the server. It should at least still be printed to standard error to help with troubleshooting users in the future